src/distributions/bernoulli.rs

```diff
@@ -18,7 +18,7 @@ use distributions::Distribution;
 /// ```
 #[derive(Clone, Copy, Debug)]
 pub struct Bernoulli {
-    /// Probability of success.
+    /// Probability of success, relative to the maximal integer.
```

Comment:
Meaning what exactly?

I note that even though this is is a "doc" comment it's not on a public field.